src/main/java/dbService/dao/UsersDAO.java

```diff
@@ -0,0 +1,38 @@
+package dbService.dao;
+
+import accounts.UserProfile;
+import dbService.executor.Executor;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class UsersDAO {
+
+  private Executor executor;
+
+  public UsersDAO(Connection connection) {
+    this.executor = new Executor(connection);
+  }
+
+  public UserProfile getUserProfile(String login) throws SQLException {
+    return executor.execQuery("select * from users where user_name='" + login + "'",
```

Comment:
```suggestion
    return executor.execQuery("SELECT * FROM users WHERE user_name='" + login + "'",
```
the same with all query